Common/src/main/resources/assets/hexcasting/lang/en_us.flatten.json5

```
@@ -1281,6 +1281,9 @@
         "not_enough_iotas.title": "Not Enough Iotas",
         not_enough_iotas: "The action required more iotas than were on the stack.$(br2)Causes light gray sparks, and as many $(l:casting/influences)$(action)Garbages/$ as would be required to fill up the argument count will be pushed.",
 
+        "stack_size.title": "Too Many Iotas",
+        "stack_size": "I tried to cast a spell which exceeded the size limit of the stack.$(br2)Causes black sparks, and my spell to halt and entire stack will be replaced with a single $(l:casting/influences)$(action)Garbage/$.",
```

```suggestion
        "stack_size": "I tried to cast a spell which exceeded the size limit of the stack.$(br2)Causes black sparks, and my entire stack will be replaced with a single $(l:casting/influences)$(action)Garbage/$.",
```

(all mishaps halt the spell; other similar mishaps don't mention it, so I don't think this one needs to either)